.gitignore

```diff
@@ -5,6 +5,7 @@
 /coverage/
 /doc/
 /pkg/
+/.rubocop*
```

Comment:
This will ignore all `.rubocop` files, we only want to ignore `.rubocop-*`
For example: `.rubocop-https---raw-githubusercontent-com-ManageIQ-guides-master--rubocop-base-yml`